### ISSUE TITLE
"ssl" is an unknown property

### ISSUE THREAD

**Issue by delatorremario** [zodern/meteor-up#560]
when I run mup setup, it show "ssl" is an unknow property

**Comment by delatorremario** [comment]
My mistake :)

**Comment by brylie** [comment]
What was the problem? I am getting the same error.

**Comment by zodern** [comment]
@brylie the `ssl` property in your config is probably in the wrong location. You can look at the [example config](https://github.com/zodern/meteor-up#example-config) to see where it should be.

**Comment by brylie** [comment]
@zodern that was the case, sorry for not following up. I had `ssl` outside of the `meteor` object. The size of the config example makes it a bit hard to follow the nesting, and perhaps an unordered, nested list would more clearly document the structure.

**Comment by delatorremario** [comment]
My config mup.js

**Comment by delatorremario** [comment]
I solve, before config https://letsencrypt.org on server
ssl: {
      // Enables let's encrypt (optional)
      crt: './ssl/bundle.crt', // this is a bundle of certificates
      key: './ssl/privkey.pem', // this is the private key of the certificate
      port: 443
    }

**Comment by brylie** [comment]
I think the goal of this feature is so that [Certbot](https://github.com/certbot/certbot) will be **automatically** configured. Note also, [LetsEncrypt is deprecated in favor of Certbot](https://www.eff.org/deeplinks/2016/05/announcing-certbot-new-tls-robot).
